rules-documentation/logsource-guides/windows/service/security.md

```diff
@@ -0,0 +1,3212 @@
+
+#### Handle Manipulation
+
+- Subcategory GUID: `{0CCE9223-69AE-11D9-BED3-505054503030}`
+- Provider: `Microsoft Windows Security Auditing`
+- Channel: `Security`
+- Event Volume: `High`
+- API Mapping: [Learn More](https://github.com/jsecurity101/TelemetrySource/tree/main/Microsoft-Windows-Security-Auditing)
+- EventID(s):
+  - 4658
+  - 4690
+
+If you're using `gpedit.msc` or similar you can enable logging for this category by following the structure below
+
+```yml
+- Computer Configuration
+    - Windows Settings
+        - Security Settings
+            - Advanced Audit Policy Configuration
+                - System Audit Policies - Local Group Policy Object
+                    - Object Access
+                        - Audit Handle Manipulation
+                            - Success and Failure
+```
+
+Alternatively you can enable logging via `auditpol` using the following command(s):
+
+```powershell
+# Enable Success audit Only
+auditpol /set /subcategory:{0CCE9223-69AE-11D9-BED3-505054503030}, /success:enable
+
+# Enable both Success and Failure auditing
+auditpol /set /subcategory:{0CCE9223-69AE-11D9-BED3-505054503030}, /success:enable /failure:disable
+```
+
+If you want to learn more about this sub-category. You can do so via MSDN - [Learn More](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/audit-handle-manipulation)
+
+#### Kernel Object
+
+- Subcategory GUID: `{0CCE921F-69AE-11D9-BED3-505054503030}`
+- Provider: `Microsoft Windows Security Auditing`
+- Channel: `Security`
+- Event Volume: `High`
+- API Mapping: [Learn More](https://github.com/jsecurity101/TelemetrySource/tree/main/Microsoft-Windows-Security-Auditing)
+- EventID(s):
+  - 4656
+  - 4658
+  - 4660
+  - 4663
+
+If you're using `gpedit.msc` or similar you can enable logging for this category by following the structure below
+
+```yml
+- Computer Configuration
+    - Windows Settings
+        - Security Settings
+            - Advanced Audit Policy Configuration
+                - System Audit Policies - Local Group Policy Object
+                    - Object Access
+                        - Audit Kernel Object
+                            - Success and Failure
+```
+
+Alternatively you can enable logging via `auditpol` using the following command(s):
+
+```powershell
+# Enable Success audit Only
+auditpol /set /subcategory:{0CCE921F-69AE-11D9-BED3-505054503030}, /success:enable
+
+# Enable both Success and Failure auditing
+auditpol /set /subcategory:{0CCE921F-69AE-11D9-BED3-505054503030}, /success:enable /failure:disable
+```
+
+If you want to learn more about this sub-category. You can do so via MSDN - [Learn More](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/audit-kernel-object)
+
+#### Other Object Access Events
+
+- Subcategory GUID: `{0CCE9227-69AE-11D9-BED3-505054503030}`
+- Provider: `Microsoft Windows Security Auditing`
+- Channel: `Security`
+- Event Volume: `Low`
```

Comment:
This was one of the verbose logs in the test env. Maybe the MS recommendation is not true here^^